### DISCUSSION TITLE
API 사용 질문

### DISCUSSION THREAD

**Discussion by tjrkdgnl** [solvedac/unofficial-documentation#35] [질답] [answered]
안녕하세요. API를 사용하다가 궁금한 점이 생겨 이렇게 문의드립니다.

1. 제가 푼 문제를 모두 가져올 때 문제의 유형까지 가져오고 싶은데  불가능할까요?

2. 사용자 정보 가져오기 API에서 solvedacToken을 쿼리 값으로 넣어야하는데 어떤 값을 넣어야하는지 잘 모르겠습니다.

감사합니다.

**Comment by kiwiyou** [comment, marked as answer]
> 1. 제가 푼 문제를 모두 가져올 때 문제의 유형까지 가져오고 싶은데 불가능할까요?

`/search/problem` 엔드포인트로 푼 문제를 가져온 뒤 `/problem/lookup`을 사용하면 될 것 같습니다.

> 2. 사용자 정보 가져오기 API에서 solvedacToken을 쿼리 값으로 넣어야하는데 어떤 값을 넣어야하는지 잘 모르겠습니다.

solved.ac에서 "BOJ로 로그인" 버튼을 눌러 BOJ로 이동한 뒤, 다시 로그인 버튼을 누르면 solved.ac으로 리다이렉트됩니다. 이때 쿠키로 solvedacToken이 전송됩니다.
